app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -96,6 +110,25 @@ public void init(int position, Contribution contribution) {
         }
     }
 
+    /**
+     * Returns the image source for the image view, first preference is given to thumbUrl if that is
+     * null, moves to local uri and if both are null return null
+     *
+     * @param thumbUrl
+     * @param localUri
+     * @return
+     */
+    @Nullable
+    private String chooseImageSource(String thumbUrl, Uri localUri) {
+        String imageSource = thumbUrl;
```

Comment:
don't do this, return the value when it is correct, you don't need a variable this can all collapse into a ternary if you so choose

Reply:
Done